apis/beacon/states/committee.yaml

```diff
@@ -0,0 +1,43 @@
+get:
+  operationId: "getEpochCommittees"
+  summary: "Get all committees for epoch"
+  description: "Retrieves the committees for the given state at the given epoch."
+  tags:
+    - Beacon
+  parameters:
+    - name: state_id
+      $ref: '../../../beacon-node-oapi.yaml#/components/parameters/StateId'
+    - name: epoch
```

Comment:
Should `epoch` be required? 

The call without `epoch` can default to `get_current_epoch(state)` of the `state_id`. It might be pretty normal to do something like `/beacon/states/head/committees` and having to get the head epoch adds an extra step here

Reply:
I will check, I think path params are required by default but I can make it explicit as well